doc/developer-guide/developer-guide.rst

```diff
@@ -0,0 +1,98 @@
+.. _Developer_Guide:
+
+#########################################
+Electrometer Developer Guide
+#########################################
+
+The Electrometer is a standard salobj CSC.
+It communicates over a RS-232 serial communication.
+
```

Comment:
Would be worth linking the manual on Docushare here.

Reply:
Can I have the link?

Reply:
electrometer and photodiode stuff is in:
https://docushare.lsst.org/docushare/dsweb/View/Collection-5176